lib/local-links-manager/distributed_lock.rb

```diff
@@ -2,20 +2,34 @@
 
 module LocalLinksManager
   class DistributedLock
+    attr_accessor :lock_name
+
+    APP = "local-links-manager"
     LIFETIME = (60 * 60) # seconds
 
     def initialize(lock_name)
       @lock_name = lock_name
     end
 
     def lock(lock_obtained:, lock_not_obtained:)
-      Services.redis.lock("local-links-manager:#{Rails.env}:#{@lock_name}", life: LIFETIME) do
+      Services.redis.lock("#{APP}:#{lock_name}", life: LIFETIME, owner: "#{APP}") do
         Rails.logger.debug('Successfully got a lock. Running...')
         lock_obtained.call
       end
     rescue Redis::Lock::LockNotAcquired => e
-      Rails.logger.debug("Failed to get lock for #{@lock_name} (#{e.message}). Another process probably got there first.")
+      Rails.logger.debug("Failed to get lock for #{lock_name} (#{e.message}). Another process probably got there first.")
       lock_not_obtained.call
     end
+
+    def unlock
+      Redis::Lock.new(Services.redis, "#{APP}:#{lock_name}", owner: "#{APP}").unlock
```

Comment:
ps.  Where you have `"#{APP}"` you can just use `APP` as it's already a string, no need to interpolate to get a new one.